src/tools/sssctl/sssctl_logs.c

```diff
@@ -309,43 +309,18 @@ errno_t sssctl_debug_level(struct sss_cmdline *cmdline,
     struct debuglevel_tool_ctx *ctx = NULL;
     struct poptOption long_options[] = {
         POPT_AUTOHELP
-        {"debug", '\0', POPT_ARG_INT | POPT_ARGFLAG_DOC_HIDDEN, &pc_debug,
```

Comment:
As you removed pc_debug, please, also remove its declaration, otherwise the following warning will be added:
```
../src/tools/sssctl/sssctl_logs.c: In function ‘sssctl_debug_level’:
../src/tools/sssctl/sssctl_logs.c:304:9: warning: unused variable ‘pc_debug’ [-Wunused-variable]
     int pc_debug = SSSDBG_DEFAULT;
```